tests/suites/test_suite_ccm.data

```diff
@@ -41,9 +41,9 @@ ccm_lengths:5:10:5:18:MBEDTLS_ERR_CCM_BAD_INPUT
 CCM lengths #6 tag length not even
 ccm_lengths:5:10:5:7:MBEDTLS_ERR_CCM_BAD_INPUT
 
-CCM lengths #7 AD too long (2^16 - 2^8 + 1)
+CCM lengths #7 AD too long (2^16 - 2^8)
 depends_on:!MBEDTLS_CCM_ALT
-ccm_lengths:5:10:65281:8:MBEDTLS_ERR_CCM_BAD_INPUT
+ccm_lengths:5:10:65280:8:MBEDTLS_ERR_CCM_BAD_INPUT
```

Comment:
Not an issue in this PR: The test code shouldn't pass off a 10-byte buffer as containing 65280 bytes, but that's a preexisting issue which is not being made worse here and which Dave has filed as https://github.com/ARMmbed/mbedtls/issues/3792.